apps/finance/app/src/App.js

```diff
@@ -150,39 +176,70 @@ class App extends React.Component {
                   />
                 </EmptyScreen>
               )}
-          </AppView>
-          <SidePanel
-            opened={newTransferOpened}
-            onClose={this.handleNewTransferClose}
-            title="New Transfer"
-          >
-            <NewTransferPanelContent
-              app={app}
+            </AppView>
+            <SidePanel
               opened={newTransferOpened}
-              tokens={tokens}
-              onWithdraw={this.handleWithdraw}
-              onDeposit={this.handleDeposit}
-              proxyAddress={proxyAddress}
-              userAccount={userAccount}
-            />
-          </SidePanel>
-        </Main>
-      </PublicUrl.Provider>
+              onClose={this.handleNewTransferClose}
+              title="New Transfer"
+            >
+              <NewTransferPanelContent
+                app={app}
+                opened={newTransferOpened}
+                tokens={tokens}
+                onWithdraw={this.handleWithdraw}
+                onDeposit={this.handleDeposit}
+                proxyAddress={proxyAddress}
+                userAccount={userAccount}
+              />
+            </SidePanel>
+          </Main>
+        </PublicUrl.Provider>
+      </Root.Provider>
     )
   }
 }
 
-const Title = styled.span`
+const ResponsiveApp = props => (
+  <React.Fragment>
+    <BreakPoint to="medium">
+      <App {...props} contentPadding={0} />
+    </BreakPoint>
+    <BreakPoint from="medium">
+      <App {...props} contentPadding={30} />
+    </BreakPoint>
+  </React.Fragment>
+)
+
+const AppBarContainer = styled.div`
+  display: flex;
+  width: 100%;
+  height: 100%;
+  justify-content: space-between;
+  align-items: center;
+  justify-content: safe;
```

Comment:
Hmm, does this need to be with an [alignment keyword](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content#Formal_syntax)?